### ISSUE TITLE
Runtime error in Release build

### ISSUE THREAD

**Issue by Reza1024** [msgpack/msgpack-cli#151]
I get `MissingMetadataException` in the release build (In a Universal Windows 10 App), but it works just fine in Debug:

> Exception thrown: 'System.Reflection.MissingMetadataException' in System.Private.CoreLib.dll
> Additional information: Reflection_InsufficientMetadata_EdbNeeded: MsgPack.Serialization.SerializationContext.SerializerGetter<System.Int32>.

```c#
public class MessagePackTest
{
	public class A { public int X; }

	public static void Test()
	{
		var a = new A { X = 0};
		var serializer = SerializationContext.Default.GetSerializer<A>();
		serializer.PackSingleObject(a);
	}
}
```

**Comment by Reza1024** [comment]
It might have to do becoming .NET Native in release:
https://msdn.microsoft.com/en-us/library/dn600634(v=vs.110).aspx#Anchor_2

**Comment by yfakariya** [comment]
Thank you for reporting. 
This problem seems to be caused by same root cause for #148. `SerializerGetter` should be retired if possible.

**Comment by yfakariya** [comment]
I found that this exception caused because of runtime serializer generation, so you can avoid this with pre-generated serializer. This article should help you: https://github.com/msgpack/msgpack-cli/wiki/Xamarin-and-Unity#generate-serializers 

I also found removing `SerializerGetter` was too hard to solve in few weeks.

**Comment by yfakariya** [comment]
In 0.7.0-alpha3, this issue should be solved. Could you try it?

**Comment by yfakariya** [comment]
This issue should be solved in 0.7.
Feel free to reopen if you find any issues.
